sdcm/nemesis.py

```diff
@@ -4568,9 +4568,20 @@ def disrupt_create_index(self):
             if not ks_cf_list:
                 raise UnsupportedNemesis("No table found to create index on")
             ks, cf = random.choice(ks_cf_list).split('.')
-            column = get_random_column_name(session, ks, cf)
-            if not column:
+            all_column_names = get_column_names(session, ks, cf)
+
+            # Query the system_schema.indexes table to get a list of columns with secondary indexes
+            index_query = f"SELECT column_name FROM system_schema.indexes WHERE keyspace_name = '{ks}' AND table_name = '{cf}'"
```

Comment:
`system_schema.indexes` table has no `column_name` column:

```
CREATE TABLE system_schema.indexes (
    keyspace_name text,
    table_name text,
    index_name text,
    kind text,
    options frozen<map<text, text>>,
    PRIMARY KEY (keyspace_name, table_name, index_name)
) WITH CLUSTERING ORDER BY (table_name ASC, index_name ASC)
```

Column name you can find in "options" column:
```
cassandra@cqlsh:keyspace1> create index on test(ck);
cassandra@cqlsh:keyspace1> SELECT * FROM system_schema.indexes WHERE keyspace_name = 'keyspace1' AND table_name = 'test' ;

 keyspace_name | table_name | index_name  | kind       | options
---------------+------------+-------------+------------+------------------
     keyspace1 |       test | test_ck_idx | COMPOSITES | {'target': 'ck'}
```